### ISSUE TITLE
radioButtons don't follow input widgets naming scheme - add alias radioButtonsInput?

### ISSUE THREAD

**Issue by daattali** [rstudio/shiny#844]
I noticed that all input widgets follow a common naming scheme of `fooInput` except for `radioButtons`.   
`textInput`, `checkboxInput`, `sliderInput`, `selectInput`, etc.

The only other inputs that don't do this are action and submit buttons, but they're different anyway because they don't really have a value that gets used in the same way as other inputs.  I think it would make sense to add an alias `radioButtonsInput` and `updateRadioButtonsInput` so keep the consistent naming scheme

**Comment by daattali** [comment]
If I get the :+1: I can submit a PR for this at some point

**Comment by bborgesr** [comment]
I think an alias would be fine for this, and a plus for consistency... @wch, @jcheng5 -- was there any reason this was not implemented this way from the start?

**Comment by alandipert** [comment]
I agree this is an inconsistency, but I'm not 100% sure it makes sense to add aliases. Even new names come with a slight cost, since they intrude on the namespace of names since chosen by users and other packages that must coexist with Shiny.

That said, I think this idea has potential and merits further consideration by the team.
